.github/workflows/ci.yaml

```diff
@@ -241,6 +265,7 @@ jobs:
         with:
           context: .
           push: true
+          platforms: linux/amd64,linux/arm64
```

Comment:
expand to a variable e.g `DOCKER_PLATFORMS` ?

Reply:
Makes sense. I moved the supported platforms into `DOCKER_TARGET_PLATFORMS`